.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
```

Comment:
I removed `0.11` from this, as it's unstable and `0.12` is released anyway.
More interesting to use `iojs`, maybe?